Doc/library/venv.rst

```diff
@@ -117,7 +117,7 @@ should be runnable without activating it.
 
 In order to achieve this, scripts installed into virtual environments have
 a "shebang" line which points to the environment's Python interpreter,
-i.e. :samp:`#!/{<path-to-venv>}/bin/python`.
+that is :samp:`#!/{<path-to-venv>}/bin/python`.
```

Comment:
```suggestion
:samp:`#!/{<path-to-venv>}/bin/python`.
```

Wouldn't it be even simpler without the extra words; that is, just stating the Python interpreter path directly?